main.py

```diff
@@ -1,11 +1,29 @@
 from flask import Flask
+from flask import send_from_directory
+from werkzeug.routing import BaseConverter
 
-app = Flask(__name__)
 
-@app.route('/')
-def index():
-    return 'Index Page'
+app = Flask(__name__, static_url_path="", static_folder="scs-app/dist/scs-app/")
 
-@app.route('/hello')
+class RegexConverter(BaseConverter):
```

Comment:
What is the regex converter for? I think it would be good we comment in the code what is going on or put it in the issue description :)